docs/make.jl

```diff
@@ -38,7 +38,7 @@ Documenter.makedocs(
     format = Documenter.HTML(
         prettyurls = get(ENV, "CI", nothing) == "true",
         ansicolor = true
-    );
+    ),
     draft
```

Comment:
Does this not break??

Reply:
It works because semicolon has been moved to the top of the keyword argument list.